### DISCUSSION TITLE
对章节字体的疑问

### DISCUSSION THREAD

**Discussion by HT0258** [sikouhjw/gdutthesis#31] [Q&A] [answered]
> 当[撰写规范](https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073)与[广东工业大学研究生学位论文撰写规范格式模板](https://yjs.gdut.edu.cn/guangdonggongyedaxueyanjiushengxueweilunwenzhuanxieguifangeshimoban.docx)矛盾时，一般以撰写规范为准。

更何况这里并没有矛盾，你说的
> 三级标题字号与正文一致，Latex中好像不一致，并且字体应该都是宋体；

是师兄师姐给的『祖传』模板吧

_Originally posted by @sikouhjw in https://github.com/sikouhjw/gdutthesis/issues/14#issuecomment-1019796711_

**Comment by sikouhjw** [comment]
有什么问题吗？

**Comment by HT0258** [comment]
> > 当[撰写规范](https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073)与[广东工业大学研究生学位论文撰写规范格式模板](https://yjs.gdut.edu.cn/guangdonggongyedaxueyanjiushengxueweilunwenzhuanxieguifangeshimoban.docx)矛盾时，一般以撰写规范为准。
> 
> 更何况这里并没有矛盾，你说的
> 
> > 三级标题字号与正文一致，Latex中好像不一致，并且字体应该都是宋体；
> 
> 是师兄师姐给的『祖传』模板吧
> 
> _Originally posted by @sikouhjw in [#14 (comment)](https://github.com/sikouhjw/gdutthesis/issues/14#issuecomment-1019796711)_

一、字体和字号
对于第三级标题字号问题，我查看了一下.cls文件，里面这部分代码如下：
%% 定义双语章节
\let\mychapter=\chapter  %这里定义的第一级标题
\let\mysection=\section   %第二级标题
\let\mysubsection=\subsection  %第三级标题
\def\gdutchapter#1#2{\enSectionTitle{#2}\mychapter{#1}}  %改变定义名称后的第一级标题
\def\gdutsection#1#2{\enSectionTitle{#2}\mysection{#1}}  %第二级标题
\def\gdutsubsection#1#2{\enSectionTitle{#2}\mysubsection{#1}}  %第三级标题

第863行代码：
  subsection = {
    format      = \zihao{4}\sffamily\raggedright,%这应该是第三级标题，这里用的4号字体，规范中是小四，这里应该为-4，上面的标题字号用该也依次往前推
    aftername   = {~},
    beforeskip  = 1.167\baselineskip-1.167\ccwd,
    afterskip   = 1.167\baselineskip-1.167\ccwd,
    fixskip     = true,

各章题序及标题           小二号黑体：
各节的一级题序及标题     小三号黑体；
各节的二级题序及标题     四号黑体；
各节的三级题序及标题     小四号黑体；

**Comment by sikouhjw** [comment, marked as answer]
你的理解是有问题的
1. 各章题序对应 `\chapter`
2. 各节的一级题序对应 `\section`
3. 各节的二级题序对应 `\subsection`
4. 各节的三级题序对应 `\subsubsection`

**Comment by HT0258** [comment]
嗯嗯，懂了，谢谢
